scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+#crm-container {
+  input[type=radio]:not(.old-radiocheckbox-style):checked+label[for]::after {
+    top: 7px;
+  }
+
+  button {
+    background: $brand-primary;
+    border: 0;
+    border-radius: $border-radius-base;
+    color: $crm-pure-white-color;
+    font-family: $font-family-base;
+    height: 36px;
+    line-height: 1;
+    padding: 11px 12px;
+    text-shadow: none;
+    text-transform: uppercase;
+    vertical-align: middle;
+
+    &:hover {
+      background: darken($brand-primary, $crm-darken-percentage);
+    }
+
+    &:active,
+    &:focus {
+      outline: none;
+    }
+
+    &.cancel {
+      background: $crm-white;
+      border: 1px solid $crm-copy;
+      color: $crm-cancel-color;
+
+      &:hover {
+        background: $crm-copy;
+        color: $crm-white;
+      }
+    }
+  }
+
+  .crm-actions-ribbon {
+    margin-bottom: 20px;
+  }
+
+  .crm-form-block.crm-form-block>.crm-submit-buttons,
+  .crm-form-block .form-item>.crm-submit-buttons {
+    border-top: 0;
+    margin-top: 0 !important;
+  }
+
+  #newLocation {
+    padding: 0 20px 20px 15px;
+
+    #crm-submit-buttons {
+      padding-left: 5px;
+    }
+
+    .crm-event-is_show_location br {
+      display: none;
+    }
+
+    h3 {
+      @include section-title;
+      margin: 0 -20px 20px -15px;
```

Comment:
Why this needs margin in minus?

Reply:
To have title full container width
![prevent-title-empty-space](https://user-images.githubusercontent.com/3272950/34294937-7980ede8-e713-11e7-811f-f47ae4cfa172.png)

Reply:
well can we make it generic then? right now you are doing it only for the `#newLocation` element, but what if we'll need to do it somewhere else in the future?